java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -4814,12 +4921,20 @@ open class KotlinFileExtractor(
         }
     }
 
-    private inner class DeclarationStackAdjuster(declaration: IrDeclaration): Closeable {
+    private inner class DeclarationStackAdjuster(val declaration: IrDeclaration, val overriddenAttributes: OverriddenFunctionAttributes? = null): Closeable {
```

Comment:
It would probably be cleaner to have another constructor, which takes an `IrFunction` and an `OverriddenFunctionAttributes`.